qa/automation/ScoringPortal.Tests.Scenarios/Sign in with Google account.feature

```diff
@@ -0,0 +1,36 @@
+Feature: 1. Sign in with Google Account
+
+   Background:
+      Given the user has a Google account
+
+   Scenario: 1.1. The application link can be opened
+      Given user has a link for the application
+      When user clicks on the link
+      Then the page should be opened
```

Comment:
May be it'll be better to give more detail what page should be opened, etc 'Welcome scoring page with google sign in button'

Reply:
Agree, should have made it more detailed